README.md

```diff
@@ -1,6 +1,38 @@
-# curriculum-cr-training-mobile-menu
+# Curriculum CR Training Mobile Menu
 
 
 This repo host an initial code for the tasks performed during the Code Reviewers training.
 
 Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+
+## :red_circle: Live Demo
+
+- [Preview Link](https://arthurgc.github.io/CodeR_Mobile_Menu)
+
+## :hammer: Built With
+- Some frameworks and technologies.
+
+
+## :blue_book: Learning Objectives
+- Learning ...........
```

Comment:
- Good Job making changes from the last reviewer.
  * Your readme file can improve a little bit better if we name the frameworks and technologies as well as in the "Learning Objectives" we can name those objectives you were working on.